lib/assets.js

```diff
@@ -110,14 +110,13 @@ const formatAssets = (includeContent, isStaticClient, asset) => {
   const assetPath = path.join(assetPathRoot, asset.name);
   const stats = fs.statSync(assetPath);
   const content = fs.readFileSync(assetPath, readOptions);
-  const hash = getHash(content, 'md5', 'hex');
 
   let name;
 
   if (isStaticClient) {
     name = asset.version + '/' + parsed.base;
   } else {
-    name = parsed.name + '.' + hash + parsed.ext;
+    name = parsed.name + '.' + parsed.ext;
```

Comment:
A byproduct of changing this for all non-static deployments will be as each SPA releases, all assets will now have a different filename and make the previous cached versions invalid.

The system should still completely work, just wanted to call out the behavior this will cause.

Reply:
Talked to Paul; we're going to make this opt-in.

Reply:
Done!